python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -998,6 +999,129 @@ def test_composite_channel_multi_output(self, ray_start_regular_shared):
         compiled_dag.teardown()
 
 
+def test_driver_and_actor_as_readers(ray_start_cluster):
+    a = Actor.remote(0)
+    b = Actor.remote(10)
+    with InputNode() as inp:
+        x = a.inc.bind(inp)
+        y = b.inc.bind(x)
+        dag = MultiOutputNode([x, y])
+
+    with pytest.raises(
+        ValueError,
+        match="DAG outputs currently cannot be read by other actor tasks.",
```

Comment:
Can you clarify what "other actor tasks" mean? Maybe improve the exception message.

Reply:
Done.